packages/babel-helper-module-transforms/src/rewrite-live-references.ts

```diff
@@ -355,31 +355,33 @@ const rewriteReferencesVisitor: Visitor<RewriteReferencesVisitorState> = {
   ) {
     const { scope, node } = path;
     const { left } = node;
-    const { exported, scope: programScope } = this;
+    const { exported, imported, scope: programScope } = this;
 
     if (!t.isVariableDeclaration(left)) {
-      let didTransform = false;
-      const bodyPath = path.get("body");
-      const loopBodyScope = bodyPath.scope;
+      let didTransformExport = false,
+        importConstViolationName;
+      const loopBodyScope = path.get("body").scope;
       for (const name of Object.keys(t.getOuterBindingIdentifiers(left))) {
-        if (
-          exported.get(name) &&
-          programScope.getBinding(name) === scope.getBinding(name)
-        ) {
-          didTransform = true;
-          if (loopBodyScope.hasOwnBinding(name)) {
-            loopBodyScope.rename(name);
+        if (programScope.getBinding(name) === scope.getBinding(name)) {
+          if (exported.get(name)) {
```

Comment:
Q: Is `exported.has(name)` sufficient here?

Reply:
Yes, you're right. Have pushed a commit making that change.